README.md

```diff
@@ -37,10 +37,12 @@ Setup and/or run CBench and/or OpenDaylight.
 OPTIONS:
     -h Show this message
     -v Output verbose debug info
-    -c Install CBench
+    -y Install CBench in Fedora
+    -a Install CBench in Ubuntu
     -t <time> Run CBench for given number of minutes
     -r Run CBench against OpenDaylight
-    -i Install OpenDaylight Helium 0.2.1
```

Comment:
Same feedback as for the `-c` flag. I suggest keeping the `-i` flag and just using `-f`/`-u` to know which OS logic to use.